docs/horizen_eon/02-connect/04-wrap_zen.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Wrapped ZEN (WZEN)
+---
+
+ZEN is the cryptocurrency or native token the Horizen blockchain and network uses for payments and as a reward for verifying transactions. However, other blockchain ecosystems, decentralized finance (DeFi), and decentralized applications (dApps) support the ERC-20 standard and not ZEN. ERC-20 is the standard for creating smart contracts-enabled fungible tokens for Ethereum-based ecosystems.
+
+Therefore, to make ZEN usable in DeFi or in other blockchain ecosystems supporting ERC-20 standards, it must be _wrapped_. Wrapped ZEN (WZEN) is a way to exchange the native cryptocurrency ZEN, for an ERC-20 compatible version of ZEN, which ultimately will allow for a variety of use cases, like swapping one token for another. Swapping of ZEN for WZEN occurs through a smart contract or code on the blockchain, which can store or send funds.
+
+WZEN is an ERC-20 token standard-compliant cryptocurrency that retains the value of ZEN. WZEN can’t be used to pay gas fees on the Horizen blockchain network but has a broader range of use cases in DeFi and dApps, from swaps to DAO voting. 
+
+
+## Smart Contract to Wrap ZEN
+
+Use the following smart contract address to wrap ZEN: \
+ \
+0xF5cB8652a84329A2016A386206761f455bCEDab6

```

Comment:
Is this meant to be a link?

Reply:
https://eon-explorer.horizenlabs.io/address/0xF5cB8652a84329A2016A386206761f455bCEDab6

Reply:
This is an address provided by Victor via John C and I believe that you plug this address into Block Explorer to access the smart contract.

Reply:
I'm not sure what the `\\` means. I say let's link it to the explorer directly here.

Reply:
not sure where the \\ came from since I did not put it there...I used the Markdown Extension in Google Doc, which probably inserted it during the conversion. Also, I only have a Pre-Gobi Blockscout...and does not work.

Reply:
I just deleted the // in the markdown file

Reply:
Yes, the link worked for me. Thanks. Will place it in the markdown file.

Reply:
One more comment: let's keep it on the same line

Reply:
agree